docs/guides/SLPDB-usage.md

```diff
@@ -0,0 +1,312 @@
+# SLPDB usage
+
+For those of you not already familiar with [SLPDB](/tooling/slpdb), it's purpose is storing all data related to the Simple Ledger Protocol, with real-time transaction and block notifications.
+
+As such, it's an invaluable tool whenever looking to build services that require reliable data on SLP specifics without needing to transverse the blockchain ourselves.
+
+## Getting started
+
+There are several SLPDB instances available for the public to query, like [https://slpdb.fountainhead.cash](https://slpdb.fountainhead.cash) or [https://slpdb.bitcoin.com](https://slpdb.bitcoin.com).
+If you are thinking that you might need your own installation of SLPDB, check our recommendations [here](/tooling/slpdb/#do-you-need-to-install-slpdb).
+
+Once you've decided on which option works best for you, the first thing you need to know is how to start querying away.
+
+Queries into SLPDB data are made using [bitquery](https://docs.bitdb.network/docs/query_v3#2-what). You can do further reading on the operators following the link, but the basic format is the following:
+
+![Bitquery format](./images/ql3.png "Bitquery format")
+
+Our documentation on [SLPDBs MongoDB Collections & Data Schema](/tooling/slpdb/#mongodb-collections-data-schema) will get you acquainted with the different datasets you can  and their field structure. At a quick glance, 
+these are the four available DBs:
+
+- `t` stands for `tokens` and contains metadata and statistics about each token.
+- `c` stands for `confirmed` and includes any confirmed SLP transaction, valid or invalid.
+- `u` stands for `unconfirmed` and includes any unconfirmed SLP transaction, valid or invalid.
+- `g` stands for `graphs` and contains an item for each valid SLP transaction.
+
+## Execution
+
+To get started you can use two basic ways to query the existing SLPDB instances: browser and node.js.
+
+### Browser
+
+If you visit [https://slpdb.fountainhead.cash](https://slpdb.fountainhead.cash) you can edit the query on the top of the page and hit the `Run query` button on the right side. This will execute your request in 
+real time and show the result on the corresponding DB result section underneath.
+
+The notation for browser and for JavaScript are a bit different: make sure to use double quote marks instead of single ones, and remove any commas before closing curly brackets.
+
+### Node.js
+
+For a more programmatic approach, we will go through how to use Node.js to execute JavaScript snippets.
+
+First off, you'll need to use npm to install the required library `node-fetch`
+
+```bash
+> npm install node-fetch
+```
+
+After that, create a new JavaScript file in the same folder tree where you installed `node-fetch`. We called this file `SLPDB.js` and it contains the following
+
+```js
+const fetch = require('node-fetch');
+
+const fetch_retry = (url, options, n=5) => fetch(url, options).catch(function(error) {
```

Comment:
Maybe a note on how its good to retry in case of network connectivity issues with 3rd party server

Reply:
Wasn't sure if I wanted to replicate the comments on retry from the original /tooling/slpdb_examples/#how-to-query-slpdb linked below. 

Added.